core/src/main/kotlin/org/evomaster/core/problem/rest/service/RestResourceFitness.kt

```diff
@@ -68,8 +69,11 @@ class RestResourceFitness : AbstractRestFitness() {
 
         doMongoDbCalls(individual.seeInitializingActions().filterIsInstance<MongoDbAction>(), actionResults)
 
-        val cookies = getCookies(individual)
-        val tokens = getTokens(individual)
+        //val cookies = AuthUtils.getCookies(client, getBaseUrl(), individual)
+        //val tokens = AuthUtils.getTokens(client, getBaseUrl(), individual)
```

Comment:
remove commented out code

Reply:
Hi Andrea,

Thanks for your comment. I have removed those comments.